src/en/MysticalMerries.lua

```diff
@@ -0,0 +1,64 @@
+-- {"id":4221,"ver":"2.3.2","libVer":"1.0.0","author":"MechTechnology","dep":["Madara>=2.3.0"]}
+
+return Require("Madara")("https://mysticalmerries.com", {
+	id = 4221,
+	name = "MysticalMerries",
+	imageURL = "https://github.com/MechTechnology/extensions/raw/dev/icons/MysticalMerries.png",
```

Comment:
Looks like you forgot to port that commit over

Reply:
Ah yes my bad :/